PORTING-ADDONS-TO-V2.md

```diff
@@ -43,11 +43,25 @@ The steps:
 
    - yes, this means you will have an `addon/addon` directory. This looks silly, but it will go away when we finish porting the addon to v2.
 
+1. Move CHANGELOG.md, LICENSE.md, and README.md files into `_addon`
+
+   In doing so, the three files will be picked up by npm pack and npm publish respectively,
+   so that the npmjs.com listing shows the proper README.md instead of an error message about missing readme,
+   and so that these files are also available in the users `node_modules`.
+
+1. Adds symlinks to `.md` files in the root to the `addon` folder
```

Comment:
Hmm. Doesn't this break the GitHub experience though? IIRC, GitHub doesn't render a `README.md` if it's a symlink (and also clicking on a symlink in GitHub doesn't take you to the file, it shows you a text file of what that symlink points to).

Reply:
> Hmm. Doesn't this break the GitHub experience though? IIRC, GitHub doesn't render a `README.md` if it's a symlink (and also clicking on a symlink in GitHub doesn't take you to the file, it shows you a text file of what that symlink points to).

@rwjblue it's "no" and "yes":
* GitHub does properly render `README.md` on the home page of repo even if `README.md` is a symlink to `addon/README.md`
* but if you open/navigate to the file in GitHub, it just renders the symlink path.

e.g. you can check https://github.com/ember-animation/ember-animated-tools and https://github.com/ember-animation/ember-animated-tools/blob/master/README.md

So `README.md` does not seem to be a real problem.

Not sure people often open `LICENSE.md` so I think it's also not real problem.

The most problematic is `CHANGELOG.md` as I think people often may want to see it.
So it's probably a question of what we'd like to achieve openable `CHANGELOG.md` in the root of the repo or `CHANGELOG.md` included in .tgz and available in `node_modules`